python_app/models.py

```
@@ -0,0 +1,64 @@
+
+
+@dataclass
+class SensorData:
+    """Container for beacon sensor readings."""
+
+    temperature: Any = None
+    pressure: Any = None
+    humidity: Any = None
+    battery_level: Any = None
+
+    def to_dict(self) -> Any:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `to_dict`

(ANN401)